src/features/Folding.ts

```diff
@@ -1,15 +1,15 @@
 /*---------------------------------------------------------
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
-
+import fs = require("fs");
```

Comment:
I think most of the rest of this codebase has a blank line after the copyright.  Other than that LGTM.  Thanks for fixing this!!